### ISSUE TITLE
What happened to the original?

### ISSUE THREAD

**Issue by Moult** [Moult/bspy#1]
Ping @stevenkfirth - I've mirrored the bspy repo here. Do you by any chance know what happened to the original? Has development stopped?

Cheers!

**Comment by stevenkfirth** [comment]
Yes, development has stopped for the moment on bspy. Which part were you interested in? The gbxml part is now here: https://github.com/building-energy/gbxml

I'm also working on a Python wrapper for EnergyPlus here: https://github.com/stevenkfirth/eprun
